org.bitfighter.Bitfighter.yaml

```diff
@@ -11,6 +11,7 @@ finish-args:
   - --share=ipc
   - --socket=x11
   - --socket=pulseaudio
+  - --device=all
```

Comment:
remove line 9 and you should be ok. That's the linter error.

Reply:
Ah, thanks for the eyes @hfiguiere. I made the swap and added a comment in case we ever get a more narrow option.